libraries/helpers.rb

```diff
@@ -53,11 +53,11 @@ def version_env_key(version)
       DeliverySugarExtras::Helpers.version_env_key(version)
     end
 
-    def add_all_change_data_to_node(node)
+    def add_all_change_data_to_node
```

Comment:
So `DSL` module is always in the context of a Chef run but `Helper` module isn't?